src/date/format-properties.js

```diff
@@ -208,6 +209,56 @@ return function( pattern, cldr ) {
 
 			// Zone
 			case "z":
+
+				// z...zzz: "{shortRegion}", eg. "PST" or "PDT".
+				// zzzz: "{regionName} {Standard Time}" or "{regionName} {Daylight Time}",
+				// eg. "Pacific Standard Time" or "Pacific Daylight Time".
+				if ( properties.tzId ) {
+
+					// The latest metazone data of the metazone array.
+					//TODO expand to support the historic metazones based on the given date.
+					var metaZone = cldr.supplemental(
+						[ "metaZones/metazoneInfo/timezone/" + properties.tzId, 0 ]
+						);
+					if ( metaZone ) {
+						properties.exemplarCity = cldr.main(
+							"dates/timeZoneNames/zone/" +
+							properties.tzId +
+							"/exemplarCity"
+							);
+						if ( length < 4 ) {
+							properties.standardTzName = cldr.main(
+								"dates/timeZoneNames/metazone/" +
+								metaZone.usesMetazone._mzone +
```

Comment:
If you still have to access the `.usesMetazone._mzone` structure, on `metaZone` assignment above you can lookup up to that point.